partials/layout-head.htm

```diff
@@ -25,6 +25,5 @@
 <![endif]-->
 
 <script type="text/javascript" src="{{ 'javascript/combined.min.js'|theme_resource }}"></script>
-<script type="text/javascript" src="{{ 'javascript/combined-source/jquery.sharrre-1.3.4.js'|theme_resource }}"></script>
```

Comment:
Yup, it mentions it in the comment. It was never connected to begin with. And changing the name made it connect causing some odd functionality.